lab_2_keywords_cooccurrence/main.py

```diff
@@ -40,7 +106,16 @@ def calculate_frequencies_for_content_words(candidate_keyword_phrases: KeyPhrase
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not correct_types(candidate_keyword_phrases, list) or not candidate_keyword_phrases:
```

Comment:
doesn't `correct_types` inlude checking for whether the object is empty ?

Reply:
yes, it does.  So?